etc/kayobe/stackhpc.yml

```diff
@@ -115,6 +119,64 @@ stackhpc_repo_rocky_appstream_version: "{{ stackhpc_repo_distribution }}"
+stackhpc_repo_centos_stream_9_opstools_version: "{{ stackhpc_repo_distribution }}"
+
+# CentOS Stream 9 - Ceph Quincy
+stackhpc_repo_centos_stream_9_storage_ceph_quincy_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/centos/9-stream/storage/x86_64/ceph-quincy/"
+stackhpc_repo_centos_stream_9_storage_ceph_quincy_version: "{{ stackhpc_repo_distribution }}"
+
+# CentOS Stream 9 Docker CE
+stackhpc_repo_centos_stream_9_docker_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/docker-ce/centos/9/x86_64/stable/"
+stackhpc_repo_centos_stream_9_docker_version: "{{ stackhpc_repo_distribution }}"
+
+# TreasureData 4 for RHEL 9
+stackhpc_repo_rhel_9_treasuredata_4_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/treasuredata/4/redhat/9/x86_64/"
+stackhpc_repo_rhel_9_treasuredata_4_version: "{{ stackhpc_repo_distribution }}"
+
+# MariaDB 10.6 for RHEL 9
+stackhpc_repo_rhel_9_mariadb_10_6_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/mariadb-10.6/yum/rhel/9/x86_64/"
+stackhpc_repo_rhel_9_mariadb_10_6_version: "{{ stackhpc_repo_distribution }}"
+
+# InfluxDB for RHEL 9
+stackhpc_repo_rhel_9_influxdb_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/influxdb/rhel/9/x86_64/stable/"
+stackhpc_repo_rhel_9_influxdb_version: "{{ stackhpc_repo_distribution }}"
+
+# MLNX OFED 5.9-0.5.6.0 RHEL 9.1
+stackhpc_repo_rhel_9_mlnx_ofed_5_9_0_5_6_0_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/mlnx_ofed/5.9-0.5.6.0/rhel9.1/x86_64/"
+stackhpc_repo_rhel_9_mlnx_ofed_5_9_0_5_6_0_version: "{{ stackhpc_repo_distribution }}"
+
+# Rocky 9 AppStream
+stackhpc_repo_rocky_9_appstream_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rocky/9/AppStream/x86_64/os/" # Should these be 9 or 9.1?
+stackhpc_repo_rocky_9_appstream_version: "{{ stackhpc_repo_distribution }}"
+
+# Rocky 9 BaseOS
+stackhpc_repo_rocky_9_baseos_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rocky/9/BaseOS/x86_64/os/" # Should these be 9 or 9.1?
+stackhpc_repo_rocky_9_baseos_version: "{{ stackhpc_repo_distribution }}"
+
+# Rocky 9 CRB
+stackhpc_repo_rocky_9_crb_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rocky/9/CRB/x86_64/os/" # Should these be 9 or 9.1?
+stackhpc_repo_rocky_9_crb_version: "{{ stackhpc_repo_distribution }}"
+
+# Rocky 9 extras
+stackhpc_repo_rocky_9_extra_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rocky/9/extras/x86_64/os/" # Should these be 9 or 9.1?
+stackhpc_repo_rocky_9_extra_version: "{{ stackhpc_repo_distribution }}"
+
+# Rocky 9 highavailability
+stackhpc_repo_rocky_9_highavailability_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rocky/9/highavailability/x86_64/os/" # Should these be 9 or 9.1?
+stackhpc_repo_rocky_9_highavailability_version: "{{ stackhpc_repo_distribution }}"
+
+# EPEL 9
+stackhpc_repo_epel_9_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/epel/9/Everything/x86_64/"
+stackhpc_repo_epel_9_version: "{{ stackhpc_repo_distribution }}"
```

Comment:
These version variables have overrides in `etc/kayobe/environments/ci-{aio,builder,multinode}/stackhpc-ci.yml`.

Reply:
I've updated them. I think I got the formatting right but can't be 100% sure